api/src/core/ports/GetSoftwareExternalData.ts

```diff
@@ -29,6 +29,11 @@ export type SoftwareExternalData = {
     sourceUrl: string;
     documentationUrl: string;
     license: string;
+    softwareVersion: string;
+    keywords: string[];
+    programmingLanguage: string[];
+    applicationCategory: string[];
```

Comment:
pluriel à mettre je pense ?
```suggestion
    programmingLanguages: string[];
    applicationCategories: string[];
```